app/controllers/posts_controller.rb

```diff
@@ -60,6 +61,25 @@ def destroy
     redirect_to council_posts_path(@council)
   end
 
+  def show_permissions
+    @posts = Post.all
+  end
+
+  def edit_permissions
+    @permissions = Permission.all
+    @post_permissions = @post.permissions.collect{|p| p.id}
```

Comment:
Prefer `map` over `collect`.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.<br>Pass `&:id` as an argument to `collect` instead of a block.